pkg/awsvpc/examples/out/explain_out/subnet_to_subnet_all_vpcs_explain_detail.txt

```diff
@@ -22,21 +22,32 @@ Path is disabled; The relevant rules are:
 	Ingress:
 		network ACL acl1 allows connection with the following allow rules
 			ruleNumber: 20, action: allow, direction: inbound, cidr: 10.240.32.0/19, protocol: all
-		security group GroupId:15 allows connection with the following allow rules
-			Inbound index: 0, direction: inbound, target: 0.0.0.0/0, protocol: udp, dstPorts: 0-65535
+		security group GroupId:9 allows connection with the following allow rules
+			Inbound index: 0, direction: inbound, target: 10.240.0.0/18, protocol: all
+
+TCP response is enabled; The relevant rules are:
+	Egress:
+		network ACL acl1 allows connection with the following allow rules
+			ruleNumber: 20, action: allow, direction: outbound, cidr: 10.240.32.0/19, protocol: all
+
+	Ingress:
+		network ACL acl1 allows connection with the following allow rules
+			ruleNumber: 20, action: allow, direction: inbound, cidr: 10.240.32.0/19, protocol: all
 
 ------------------------------------------------------------------------------------------------------------------------
 
-Connections from r1[10.240.48.198] to q2[10.240.32.122]: protocol: TCP dst-ports: 9080
+No connections from r1[10.240.48.198] to q1[10.240.32.91];
+	connection is blocked since ingress's and egress's connections do not intersect.
+	egress connection: protocol: TCP dst-ports: 9080, ingress connection: protocol: UDP
+
+Egress: security group GroupId:22 allows connection; network ACL acl1 allows connection
+Ingress: network ACL acl1 allows connection; security group GroupId:15 allows connection
```

Comment:
```suggestion
No connectivity from r1[10.240.48.198] to q1[10.240.32.91];
	connectivity is blocked since traffic patterns allowed at ingress are disjoint from the traffic patterns allowed at egress.
	allowed egress traffic: protocol: TCP dst-ports: 9080, allowed ingress traffic: protocol: UDP

Egress: security group GroupId:22 allows connection; network ACL acl1 allows connection
Ingress: network ACL acl1 allows connection; security group GroupId:15 allows connection
```

Reply:
done